examples/gno.land/r/docs/img_embed/img_embed.gno

```diff
@@ -0,0 +1,10 @@
+package image_embed
+
+// Render displays a title and an embedded image from Imgur
+func Render(path string) string {
+	return `# Image Embed Example
+
+Here’s an example of embedding an image in a Gno realm:
+
+![Example Image](https://i.imgur.com/So4rBPB.jpeg)`
```

Comment:
@jefft0, I don't remember this.

Our goal is to create a list of whitelisted domains for auto-preview. Imgur is, if I recall correctly, moderated. Therefore, Gnoweb can be updated to disable automatic previews for non-whitelisted domains while still allowing links to any URL, whether for images or other content.

Note: This example aims to discourage users from uploading all their files in base64 on the chain. Such cases should be very advanced and only relevant in rare situations. By default, image embedding or linking should be the preferred method for sharing assets.

Reply:
can't wait for gnoweb refacto so we can add `ipfs://` embed support :)

Reply:
I will likely address all the bad practices I see everywhere by providing examples here. Here’s my shortlist:

-  Base64 meme images (inefficiency)
-  Writing JSON encoders (makes code hard to read, mixes concepts and targets)
-  Creating top-level functions for everything instead of promoting `maketx run`

I will write new examples soon and open issues so we can update the tools (e.g., `gnoweb`).